pkgs/racket-doc/scribblings/reference/evts.scrbl

```diff
@@ -307,13 +307,15 @@ with timeouts that have not yet expired. The system-idle event's
 ]}
 
 
-@defproc[(alarm-evt [msecs real?]) evt?]{
+@defproc[(alarm-evt [msecs real?] [monotonic? boolean? #f]) evt?]{
 
 Returns a @tech{synchronizable event} that is not @tech{ready for synchronization} when
-@racket[(current-inexact-milliseconds)] would return a value that is
+@racket[(milliseconds)] would return a value that is
```

Comment:
Using `_milliseconds` here and each other use would typeset in italics, making more clear that it's a metavariable.

Reply:
Done